beetsplug/convert.py

```diff
@@ -336,14 +338,33 @@ def copy_album_art(self, album, dest_dir, path_formats, pretend=False):
                            util.displayable_path(album.artpath))
             return
 
-        if pretend:
-            self._log.info(u'cp {0} {1}',
-                           util.displayable_path(album.artpath),
-                           util.displayable_path(dest))
+        resize = False
+        maxwidth = None
+
+        if self.config['copy_album_art_maxwidth']:
+          maxwidth = self.config['copy_album_art_maxwidth'].get(int)
+          size = ArtResizer.shared.get_size(album.artpath)
+          if size:
+              resize = ArtResizer.shared.must_resize(size, maxwidth)
+
+        if resize:
+            if pretend:
+                self._log.info(u'resize_artwork {0} {1}',
```

Comment:
This can be human-readable: like "resizing artwork to" instead of "resize_artwork" (since it's not a shell command anyway).